x/warp/config.go

```diff
@@ -73,6 +74,7 @@ func (*Config) Key() string { return ConfigKey }
 
 // Verify tries to verify Config and returns an error accordingly.
 func (c *Config) Verify(precompileconfig.ChainConfig) error {
+	// TODO: return an error if Warp is enabled before DUpgrade
```

Comment:
Could we add this in this PR? It seems like a good safety check.

Reply:
I planned to, but realized it was applied in this PR: https://github.com/ava-labs/subnet-evm/pull/801/files#diff-fd0c998db0ab5b6c9249e67a71921b09d0ea7180742e064beb890fda3176cfe7. I'll make a separate PR to apply this change to rebase this off of and check `DUpgrade` here.

Reply:
https://github.com/ava-labs/subnet-evm/pull/825/files

Reply:
Merged